### ISSUE TITLE
Two Wallet Mining

### ISSUE THREAD

**Issue by Teddymeyn** [NebuTech/NBMiner#398]
Is there a way to mine using 2 different wallets at the same time ?

**Comment by pdieppa** [comment]
You can run two copies of the miner, each one with a different wallet. This should give you about 50% of the total hash rate per miner.

**Comment by Teddymeyn** [comment]
that's not stable and the miner is crashing, I am trying to understand if there is any way to allocate hash rate to different wallet

**Comment by pdieppa** [comment]
I don't think is possible with a single instance of the miner. Other option is if you have more than one GPU, you can run separate copies of the program and assign certain GPUs to different wallets.

**Comment by Teddymeyn** [comment]
Yes, this one is a good option but I have 3 GPUs and I need to divide the hash to 50%.

**Comment by pdieppa** [comment]
That's going to be hard. The other option is to hash 12 hours first wallet and then other 12 to the second wallet. Not ideal since it's hard to automate the process.

**Comment by Teddymeyn** [comment]
Since the pool that I am using PPLNS, it will make the payout a little low, while changing to both wallets it will look like I am jumping pools for better luck.... I appreciate your time to respond to this...
